source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -150,14 +150,19 @@
 
         this.evaluation.fromJs(data.Evaluation);
 
-        if (this.evaluation.readyForDevolution) {
+        if (this.evaluation.readyForDevolution && this.userView == 3) {
+            this.hasAverageColumn = false;
             _.chain(self.califications)
-            .filter(function (calification) {
-                return calification.owner == 0 || calification.owner == 3;
-            })
-            .map(function(calificationEditable){
-                calificationEditable.hasShowIcon = false;
-                calificationEditable.show(true);
+            .map(function (calification) {
+                if (calification.owner == 1 || calification.owner == 2) {
```

Comment:
calificator.owner 1 and 2 are responsible and evaluators respectively. When the evaluation is ready for devolution, these calcifications must come hide